packages/common/src/config.ts

```diff
@@ -1,17 +1,50 @@
-import { getEnv } from "./env";
+import { z } from "zod";
 
-type PinataConfig = {
-  jwt: string;
-  gateway: string;
-  pinataBaseUrl: string;
+type Config = {
+  env: string;
```

Comment:
A union here would allow to catch typos at compile time.

Reply:
good one, I updated it